types/pdfkit/pdfkit-tests.ts

```diff
@@ -304,6 +276,10 @@ doc.endMarkings();
 // structure content methods
 const structureContent2 = doc.markStructureContent('H1');
 structureContent.push(structureContent2);
+// automatic marking and structure construction for text
```

Comment:
This is where I've edited the tests.

Reply:
Looks alright, since changes must be made anyway, you could also add a case using the structTypes/structType option.